app/Http/Controllers/CampaignController.php

```diff
@@ -143,10 +144,14 @@ public function update(CampaignUpdateRequest $request, Campaign $campaign)
     /**
      * @param \Illuminate\Http\Request $request
      * @param \App\Models\Campaign $campaign
-     * @return \Illuminate\Http\Response
+     * @return \Illuminate\Http\RedirectResponse
      */
     public function destroy(Request $request, Campaign $campaign)
     {
+        if (!$campaign->is_owner) {
```

Comment:
Could/ should the `getIsOwnerAttribute()` be used here instead of accessing `is_owner` directly? If not, which context is it best suited for

Reply:
With [accessors in Laravel](https://laravel.com/docs/8.x/eloquent-mutators#defining-an-accessor), you should always be calling them by their their attribute name instead of calling the function directly. The reason for this is because laravel mostly expects you to access all data about a model through the "magic getter" method from the base eloquent model. Plus its a best practice to access it that way